tests/codegen/call_kstack.cpp

```diff
@@ -10,12 +10,7 @@ TEST(codegen, call_kstack)
 {
   auto result = NAME;
 
-  // Mode doesn't directly affect codegen, so the programs below should
-  // generate the same IR
-  test("kprobe:f { @x = kstack(); @y = kstack(6) }", result);
-  test("kprobe:f { @x = kstack(perf); @y = kstack(perf, 6) }", result);
-  test("kprobe:f { @x = kstack(raw); @y = kstack(raw, 6) }", result);
```

Comment:
curious, why remove the checks for the different stack modes?

Reply:
Because these now generate different LLVM IR (they used to generate the same) so we'd need to keep separate `.ll` files for then. It's better explained in the commit message.

Reply:
Ah, I see. It's probably an overkill, the only change will be in names of global variables for maps.